hw/ip/pattgen/data/pattgen_testplan.hjson

```diff
@@ -50,6 +50,43 @@
       milestone: V2
       tests: ["pattgen_perf"]
     }   
+    {
+      name: rand
+      desc: '''
+            Checking ip operation with random counter values
```

Comment:
Include functional cover point that rollover value is reached and counter is reset

Reply:
Could you please document this in the `covergroups` section of the `pattgen_testplan.hjson`?